### DISCUSSION TITLE
Why does classification ViT have nn.Tanh() in the end?

### DISCUSSION THREAD

**Discussion by chris-clem** [Project-MONAI/MONAI#4580] [Q&A]
Hello together,

I was wondering why does the Vision Transformer (ViT) with `classification=True` have `nn.Tanh()` in the end of the `classification_head` (https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/vit.py#L100)? The other classification models output the raw logits.

Best
Christoph

**Comment by wyli** [comment]
thanks for reporting, I create a ticket to track this issue. https://github.com/Project-MONAI/MONAI/issues/4581
